### PR TITLE
Feature profile page skeleton

### PR DESCRIPTION
**Pull request by m3tron**
Refactored dashboard with AppBar and Drawer which included friendsList.

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { MuiThemeProvider, CssBaseline } from "@material-ui/core";
 import { BrowserRouter, Route, Redirect } from "react-router-dom";
 import { theme } from "./themes/theme";
 import AuthPage from "./pages/AuthPage";
-import Dashboard from "./pages/Dashboard";
+import Dashboard from "./pages/Dashboard/index";
```
Comment (skassam21) [inline client/src/App.jsx]: let's be consistent with naming conventions - I more commonly see with React the name of the files are the name of the components and are PascalCase

File: client/src/components/AppBarDrawer/index.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+    width: drawerWidth,
+    flexShrink: 0
+  },
+  drawerPaper: {
+    width: drawerWidth
+  },
+  // content: {
+  //   flexGrow: 1,
+  //   padding: theme.spacing(3)
+  // },
```
Comment (skassam21) [inline client/src/components/AppBarDrawer/index.js]: remove commented out code

File: client/src/components/AppBarDrawer/index.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    prevOpen.current = open;
+  }, [open]);
+
+  const handleDialog = () => {
+    setOpen(!open);
+  };
+
+  return (
+    <div className={classes.root}>
+      <CssBaseline />
```
Comment (skassam21) [inline client/src/components/AppBarDrawer/index.js]: I beleive CssBaseline shouold only exist at the top level react component

File: client/src/components/FriendList/index.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { List, Typography } from "@material-ui/core";
+import Friends from "../Friends";
+
+const useStyles = makeStyles(theme => ({
+  //title: { flexGrow: 1 }
```
Comment (skassam21) [inline client/src/components/FriendList/index.js]: again commented out code

File: client/src/components/FriendList/index.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+}));
+
+const FriendList = ({ friends }) => {
+  const classes = useStyles();
+
+  return (
+    <>
+      <Typography variant="h4" className={classes.title}>
```
Comment (skassam21) [inline client/src/components/FriendList/index.js]: need some padding or margin here for the friends title

**Review by skassam21 — COMMENTED** [review]
Small comments

**Review by chungleee — APPROVED** [review]

File: client/src/App.jsx [hunk 6]
```diff
@@ -5,7 +5,7 @@ import { MuiThemeProvider, CssBaseline } from "@material-ui/core";
 import { BrowserRouter, Route, Redirect } from "react-router-dom";
 import { theme } from "./themes/theme";
 import AuthPage from "./pages/AuthPage";
-import Dashboard from "./pages/Dashboard";
+import Dashboard from "./pages/Dashboard.js";
```
Comment (k-awe-some) [inline client/src/App.jsx]: can we possibly combine the two Dashboard components into one `Dashboard.jsx` file?

**Review by k-awe-some — CHANGES_REQUESTED** [review]
Profile page looks solid yet Create new poll is not letting user upload images, code looks fine to me so I'm thinking conflicts come from `PollDialog` being used in 2 places (`AppBarDrawer` and `NavBar`) ?!

also if we could all use `.jsx` for React components that'd be more consistent.

**Pull request merged**